cni-repair-controller/integration/linkerd-cni-config.yml

```diff
@@ -0,0 +1,7 @@
+# This config adds an extra initContainer that will make linkerd-cni to delay
+# its start for 15s, so to allow time for the pause DaemonSet to start before
+# the full CNI config is ready and enter a failure mode
+extraInitContainers:
+- name: sleep
+  image: busybox
```

Comment:
on the nitpicky side, the CNI plugin runs alpine, can we re-use the same image so we don't pull busybox in tests?

Reply:
Good thinking :+1: